### PR TITLE
Gseal 56 re writer module

### PR DESCRIPTION
**Pull request by gpseal**
Re writer module to reWrite scraped articles and load them to a new table

Scraped data will now go to a table called "Article_Raw", which will not be displayed on the front end.

Once the reWriter module is executed, it will pull all articles that have not already been re-written and paraphrase them one at a time, loading them to the original "Article" table after each article has been re-written.

Execute the module by entering the reWriter folder and entering "python reWriter.py"

### CODE REVIEW

**Review by AardhynLavender — CHANGES_REQUESTED** [review]
Nice. 

Just a few things thingsin the comments bellow.


#### Formatting

Can you make sure the formatter is being run in the `/api` directory.

```shell
npm run pretty
```

Or configure your IDE to format on save if that option is available.

**Review by ArtemKech — COMMENTED** [review]
@gpseal Just a small change to implement. Can you please add `{PORT}` variable on line 52 in `reWriter/reWriter.py` file. There is a variable for a port number inside `.env` file for that. 
But overall, really good job! I am glad that you have finally found a good paraphraser that actually works. I remember how painful that was to find a free model that does the job. 
Apply this change, and it will be ready for approval.

File: api/route/Article/article.create.ts [hunk 1]
```diff
@@ -1,63 +1,48 @@
...
-    // todo: remove fake category`
-    getClient().query<Article>(insertion, [], (err, results) => {
-      if (err)
-        return Error(res, INTERNAL_SERVER_ERROR, {
-          message: err.message || 'An unknown error occurred',
-          type: 'DatabaseError',
-        });
+  const { article } = req.body as { article: Article };
+
+  const client = new Client({
```
[the remark this answers is not in the document]
Comment (gpseal) [reply api/route/Article/article.create.ts:11]: Let me know if I haven't used this correctly

File: database/src/init.sql [hunk 2]
```diff
@@ -21,6 +21,19 @@ create table Article (
...
+        category varchar(32) not null references Category(category),
+        fake_category varchar(32) not null references Category(category),
+        body json not null,
+        source_url varchar(256) not null,
+        cover_url varchar(256) null,
+        publication_date varchar(24) null, -- I don't like this, should be `Date` or `Timestamptz` type
+        retrieved_date timestamptz not null default now()
+);
+
 -- Defaults --
```
[the remark this answers is not in the document]
Comment (gpseal) [reply database/src/init.sql:37]: I have implemented this, but it was causing a side effect with the article summary route query.  I have updated this to work, but I am not overly sure that it is functioning correctly, perhaps take a quick loo at it to make sure its all in order (I guess the fake category stuff is no longer relevant so will need to be removed anyway?)

**Review by AardhynLavender — APPROVED** [review]
Fantastic

**Comment by ArtemKech** [comment]
Thanks for the applied changes as requested earlier. That looks better now! It's ready to go
<sub><sup>merry x-mas</sup></sub>

**Review by ArtemKech — APPROVED** [review]

**Pull request merged**